1

```diff
@@ -0,0 +1,32 @@
+Add / make fit for purpose email.getlist api call
```

Comment:
@eileenmcnaughton the purpose of this file is?

Reply:
I think you know the answer ... removing